common/wallet_tx.c

```diff
@@ -104,3 +166,34 @@ struct command_result *wtx_select_utxos(struct wallet_tx *tx,
 	}
 	return NULL;
 }
+
+struct command_result *wtx_fees_from_utxos(struct wallet_tx *tx,
+					u32 fee_rate_per_kw,
+					size_t out_len)
+{
+	struct command_result *res;
+	size_t weight, i;
+
+	tx->amount = AMOUNT_SAT(0);
+	/* The transaction has `tal_count(tx.utxos)` inputs and one output */
+	/* version + in count + out count + locktime    index + value + script length */
+	weight = 4 * (4 + 1 + 1 + 4) + 4 * (8 + 1 + out_len);
```

Comment:
Technically the `out_len` might be longer than `1` but this is probably fine.

```
(8 + 1 + out_len)
```

Reply:
That's why it's a variable